### DISCUSSION TITLE
Reason for max-diagnostics limit 200?

### DISCUSSION THREAD

**Discussion by MrTact** [biomejs/biome#1050] [Q&A] [answered]
I'm working on migrating a large codebase from eslint to biome, and there are WAY more than 200 diagnostics. Is there a technical reason for this limit? I'd really like to output the results to a file and use that to drive some targeted `--apply-unsafe` calls (so, for example, I can put all the `noUnnecessaryElse` diagnostic fixes into one commit, to make it easier to code review).

**Comment by ematipico** [comment, marked as answer]
Printing on the terminal is an onerous operation and it can slow down the command a lot, that's why we put a limit.

We plan to lift the limit soon

**Comment by Conaclos** [comment]
Closing because we lifted the maximum limit.
